cmd/flags/manifest.go

```diff
@@ -37,10 +37,7 @@ func newManifestsConfig(vipr *viper.Viper) (*manifestsmanager.Config, error) {
 	manifestPath := vipr.GetString(manifestPathKey)
 	_, err := os.Stat(manifestPath)
 	if err != nil {
-		if os.IsNotExist(err) {
-			return nil, fmt.Errorf("cannot read manifest path %s", manifestPath)
-		}
-		return nil, err
+		return nil, fmt.Errorf("cannot read manifest path %s: %v", manifestPath, err)
```

Comment:
As we discussed, we maybe don't need to wrap; but in the other case, you wrap.
We should decide on 1. I guess not wrapping is fine as it's really at app startup that it happens.